lua/nvim-treesitter/parsers.lua

```diff
@@ -802,6 +802,21 @@ list.prisma = {
   maintainers = { "@elianiva" },
 }
 
+list.foam = {
+  install_info = {
+    url = "https://github.com/FoamScience/tree-sitter-foam",
+    branch = "master",
+    files = { "src/parser.c", "src/scanner.cc" },
+    generate_requires_npm = true,
+  },
+  maintainers = { "@FoamScience" },
+  filetype = "foam",
```

Comment:
This line is actually not necessary when the parser name is also foam. But it also doesn't do any harm.